plugins/outputs/datadog/datadog.go

```diff
@@ -46,17 +49,15 @@ type Point [2]float64
 
 const datadog_api = "https://app.datadoghq.com/api/v1/series"
 
-func NewDatadog(apiUrl string) *Datadog {
-	return &Datadog{
-		apiUrl: apiUrl,
-	}
-}
-
 func (d *Datadog) Connect() error {
 	if d.Apikey == "" {
 		return fmt.Errorf("apikey is a required field for datadog output")
 	}
 
+	if d.URL == "" {
+		d.URL = datadog_api
```

Comment:
Set this in the init() function and then it will be overridden automatically when the plugin in loaded.